test/types/models.test.ts

```diff
@@ -205,4 +206,48 @@ function inheritance() {
       await this.findOneAndDelete({ installationId });
     }
   }
+}
+
+export async function m0_0aModel() {
+  const AutoTypeSchema = m0_0aSchema();
+  const AutoTypeModel = model('AutoTypeModel', AutoTypeSchema);
+
+  /* -------------------------------------------------------------------------- */
+  /*                            Model-functions-test                            */
+  /* -------------------------------------------------------------------------- */
+  // Create should works with arbitrary objects.
+  const randomObject = await AutoTypeModel.create({ unExistKey: 'unExistKey', description: 'st' });
+  expectType<string>(randomObject.unExistKey);
```

Comment:
This shows intellisense for both doc type and new inserted props.
@vkarpov15  What do you think about this test?